terraform.io/aws.com/ec2/main.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+      version = "~> 4.16"
+    }
+  }
+
+  required_version = ">= 1.2.0"
+}
+
+resource "aws_instance" "example_server" {
+
+  ami = "ami-098efcc0d4f80810e" # important to match ami with the region !
```

Comment:
Even better is to [lookup](https://stackoverflow.com/questions/65686821/terraform-find-latest-ami-via-data) the AMI by name in the current region.
```terraform
data "aws_ami" "something_cheap" {
  most_recent = true
  owners      = ["amazon"]

  filter {
    name   = "name"
    values = ["amzn2-ami-ecs-hvm-2.0.202*-x86_64-ebs"]
  }
}

resource "aws_instance" "example_server" {
  ami = data.aws_ami.something_cheap.id
  ... 
}
```